Automattic/jetpack
# DNA: Move Jetpack Usage tracking to its own file

This PR moves some of the tracking actions to its own file in order to reduce the overall size and responsibilities of `class.jetpack.php`

**Questions**
* We are adding a new `src` file so we can use autoloader, 👍 or 👎 ?
* Would love some review in terms of new actions and files names

**Testing Instructions**

* enable WP_DEBUG_LOG for your site
* navigate to the Jetpack Dashboard of your site
* make sure to perform actions that are tracked like activating modules
* make sure no errors or warnings are logged into the error_log
* try disconnecting and reconnecting Jetpack
* try a full sync
* try logging in with SSO
* do tests still pass?
* is your debug.log clean?
